_posts/2023-07-20-homebrew-4.1.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 4.1.0
+author: MikeMcQuaid
+redirect_from: /blog/4.1.0/
+---
+
+Today, I'd like to announce Homebrew 4.1.0. The most significant changes since 4.0.0 are significant improvements to the security/reliability/performance/usability of Homebrew 4.0.0's new JSON API, the completion of the migration of analytics from Google Analytics in the US to InfluxDB in the EU and groundwork for later macOS Sonoma (14) support.
+
+Major changes and deprecations since 4.0.0:
+
+- [`brew` downloads of formula/cask APIs use a signed API endpoint with client signature verification.](https://github.com/Homebrew/brew/pull/14733)
+- We've made many improvements around the new 4.0.0 feature of using JSON files downloaded from [formulae.brew.sh](https://formulae.brew.sh) for package installation rather than local homebrew/core and homebrew/cask taps.
+  - If you had previously set `HOMEBREW_NO_AUTO_UPDATE`, `HOMEBREW_NO_INSTALL_FROM_API` or `HOMEBREW_AUTO_UPDATE_SECS` to work around bugs or annoyances: please consider unsetting these and tweaking the values based on the new behaviour. Under some circumstances, [you may see a one-time message nudging you do to this](https://github.com/Homebrew/brew/pull/14592).
+- [`brew doctor` warns if Homebrew/homebrew-core or Homebrew/homebrew-cask seem to be tapped unnecessarily so you can `brew untap` them to save time and disk space.](https://github.com/Homebrew/brew/pull/15341)
+- [formulae.brew.sh](https://formulae.brew.sh) provides [new analytics for Homebrew Developer Configuration, OS/Architecture/CI, Homebrew Prefixes and Homebrew Versions](https://formulae.brew.sh/analytics/). This is based on existing data that was already gathered.
+- We destroyed (and did not migrate/back up) all of Homebrew's Google Analytics data on 2023-06-16 and [all Google Analytics code has been removed.](https://github.com/Homebrew/brew/pull/15550)
+- Homebrew's analytics are only sent to our InfluxDB instance hosted in the EU.
+  - If you had previously set `HOMEBREW_NO_ANALYTICS` because you didn't like Google Analytics and/or data being sent to the USA: please consider unsetting this and setting `HOMEBREW_NO_GOOGLE_ANALYTICS` instead, allowing analytics data to be sent to our new InfluxDB host. Again, under some circumstances, [you may see a one-time message nudging you do to this](https://github.com/Homebrew/brew/pull/14592).
```

Comment:
We should change “USA” to “US” for consistency.

Does `HOMEBREW_NO_GOOGLE_ANALYTICS` do anything now that the code has been removed?

Reply:
Done/fixed both!